backends/ebpf/ebpfTable.cpp

```diff
@@ -478,9 +478,14 @@ void EBPFTable::emitKey(CodeBuilder* builder, cstring keyName) {
             } else if (width <= 64) {
                 swap = "bpf_htonll";
             } else {
-                // TODO: handle width > 64 bits
-                ::error(ErrorType::ERR_UNSUPPORTED,
-                        "%1%: fields wider than 64 bits are not supported yet", fieldName);
+                // The code works with fields wider than 64 bits for PSA architecture. It is shred
+                // with filter model, so should works but has not been tested. Error message is
```

Comment:
since you are doing this you should also consider inserting the midend pass saturationElim which will enable saturated arithmetic for ebpf.

Reply:
I'm not sure if the midend pass saturationElim is related to this PR. In this PR I enable basic operations on wide fields (wider than 64 bits), like assignment, but I didn't touch the arithmetic operations.